### PR TITLE
Added known issue with MoH

### PR DESCRIPTION
**Pull request by FilippSe**
Added known issue with MoH

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-media-optimization.md [hunk 1]
```diff
@@ -331,5 +331,5 @@ The following is a list of known issues that are currently present in Local Medi
...
 | Call escalations result in dropped calls when the Teams client is identified as internal.| Disable Local Media Optimization on the Direct Routing SBC.|
 | Call escalations from 1 to 1 call between internal customers to multiparty call with external customer/resource result in dropped calls | Work in progress on a fix. Alternatively, disable Local Media Optimization on the Direct Routing SBC.|
-
+| Teams User puts call On Hold.  Music plays on PSTN end, LMO (Local Media Optimization) is working OK. Teams User resumes the call.Call to PSTN resumes fine but LMO is not working, call continues via Central (Proxy) SBC | When a user parks a call to initiate a music on hold (MoH), it is being escalated from 1:1 to a multiparty call by Call Controller to invoke Media Controller and Media Processor (serving as AVMCU mixer) through which MoH reaches a user who has been put on hold. De-escalation to a 1:1 call after un-hold never happens as per design. Disable Local Media Optimization on the Direct Routing SBC.|
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-media-optimization.md:334]:
```suggestion
| Teams user puts the call On Hold. Music plays on the PSTN end and Local Media Optimization is working. The Teams user resumes the call. The call to PSTN resumes but Local Media Optimization is not working and the call continues via Central (Proxy) SBC | When a user parks a call to initiate music on hold (MoH), it is being escalated from 1:1 to a multiparty call by the Call Controller to invoke Media Controller and Media Processor (serving as AVMCU mixer) through which MoH reaches a user who has been put on hold. De-escalation to a 1:1 call after the call resumes never happens as per design. Disable Local Media Optimization on the Direct Routing SBC.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@FilippSe - Could you please commit my suggested changes? Many thanks.

**Pull request merged**